101-load-balancer-cross-region/azuredeploy.json

```diff
@@ -0,0 +1,890 @@
+                "[resourceId('Microsoft.Network/loadBalancers', variables('lbName-r2'))]",
+                "[resourceId('Microsoft.Network/networkSecurityGroups', variables('nsgName-r2'))]"
+            ],
+            "properties": {
+                "ipConfigurations": [
+                    {
+                        "name": "ipconfig1",
+                        "properties": {
+                            "privateIPAllocationMethod": "Dynamic",
+                            "subnet": {
+                                "id": "[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('vNetName-r2'), variables('vNetSubnetName-r2'))]"
+                            },
+                            "loadBalancerBackendAddressPools": [
+                                {
+                                    "id": "[resourceId('Microsoft.Network/loadBalancers/backendAddressPools', variables('lbName-r2'), variables('lbBackendPoolName-r2'))]"
+                                },
+                                {
+                                    "id": "[resourceId('Microsoft.Network/loadBalancers/backendAddressPools', variables('lbName-r2'), variables('lbBackendPoolNameOutbound-r2'))]"
+                                }
+                            ]
+                        }
+                    }
+                ],
+                "networkSecurityGroup": {
+                    "id": "[resourceId('Microsoft.Network/networkSecurityGroups', variables('nsgName-r2'))]"
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Compute/virtualMachines/extensions",
+            "apiVersion": "2020-06-01",
+            "name": "[concat('vm-r2','-',copyIndex(1),'/','InstallWebServer')]",
+            "location": "[parameters('location-r2')]",
+            "copy": {
+                "name": "extensionCopy",
+                "count": 3
+            },
+            "dependsOn": [
+                "vmCopy"
+            ],
+            "properties": {
+                "publisher": "Microsoft.Compute",
+                "type": "CustomScriptExtension",
+                "typeHandlerVersion": "1.7",
+                "autoUpgradeMinorVersion": true,
+                "settings": {
+                    "commandToExecute": "powershell.exe Install-WindowsFeature -name Web-Server -IncludeManagementTools && powershell.exe remove-item 'C:\\inetpub\\wwwroot\\iisstart.htm' && powershell.exe Add-Content -Path 'C:\\inetpub\\wwwroot\\iisstart.htm' -Value $('Hello World from ' + $env:computername)"
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Compute/virtualMachines",
+            "apiVersion": "2020-06-01",
+            "name": "[concat('vm-r2','-',copyIndex(1))]",
+            "location": "[parameters('location-r2')]",
+            "zones": [
+                "[copyIndex(1)]"
```

Comment:
seems this template will fail if deployed to a region that doesn't support zones?

Reply:
I got this part of the template from the published template for deploying a standard load balancer which has zones.
https://docs.microsoft.com/en-us/azure/load-balancer/quickstart-load-balancer-standard-public-template

Reply:
@mumian - do you have a preference on how to handle these?  We can doc it or add allowedValues...

Reply:
The article has this part - "The template deployment creates three availability zones. Availability zones are supported only in certain regions. Use one of the supported regions. If you aren't sure, enter centralus."

I will add the description to the quickstart template.